koap/src/jsMain/kotlin/jsBridge.kt

```diff
@@ -0,0 +1,26 @@
+package com.juul.koap
```

Comment:
Rename to `Script.kt` or `Main.kt`?
Although conceptually it's a "bridge" to the transpiled Javascript, I don't think that's immediately obvious. `Main` is convention for the entry point in Java, so seems more familiar for other devs?